src/main/main.cc

```diff
@@ -151,7 +151,7 @@ bool initialize(const std::vector<RunTab> &tabs) {
 				break;
 			}
 		} catch (const std::exception &e) {
-			safs_pretty_syslog(LOG_ERR, "%s", e.what());
+			safs::log_err("unhandled exception ({}): {}", typeid(e).name(), e.what());
```

Comment:
consider the usage of the recently created `log_error_code`:
```suggestion
			safs::log_error_code(e, "unhandled exception");
```

Reply:
Doesn't work with exceptions. I've added a log_exception to slogger based on your for errno